ckan/common.py

```diff
@@ -267,6 +267,21 @@ def aslist(obj: Any, sep: Optional[str] = None, strip: bool = True) -> Any:
         return [obj]
 
 
+def clean_context(
```

Comment:
Whatever this function ends up being called, it should be defined in `ckan/logic/__init__.py` with the rest of the actions/context logic

Reply:
And please add it to the toolkit as well, as in [here](https://github.com/ckan/ckan/blob/dc49cece33e447858d0008bc71b9a4ab724602ee/ckan/plugins/toolkit.py#L20)